cpp/ql/test/experimental/query-tests/Security/CWE/CWE-193/pointer-deref/test.cpp

```diff
@@ -254,3 +254,49 @@ void test17(unsigned *p, unsigned x, unsigned k) {
         p[i] = x; // GOOD [FALSE POSITIVE]
     }
 }
+
+struct array_with_size
+{
+  int *xs;
+  unsigned len;
+};
+
+void test17(unsigned len, array_with_size *s)
```

Comment:
The `s` argument does not seem to be used and similar for the other functions below.

Reply:
Thanks. I managed to reduce the issue even further, but forgot to remove the (now superfluous) parameter.